lighthouse-core/test/fixtures/artifacts/perflog/defaultPass.trace.json

```diff
@@ -0,0 +1,14 @@
+[
```

Comment:
copied over from `fixtures/traces/trace-user-timings.json` cause it was small and we really only need to test the loading part :)